lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -4435,10 +4435,6 @@ ConformanceChecker::resolveWitnessViaLookup(ValueDecl *requirement) {
           requiredAccessScope.requiredAccessForDiagnostics();
         auto proto = conformance->getProtocol();
         auto protoAccessScope = proto->getFormalAccessScope(DC);
-        // Skip diagnostics of a witness of a package protocol that is inlinalbe
-        // referenced in an interface file.
-        if (proto->skipAccessCheckIfInterface(DC, requiredAccess, protoAccessScope))
```

Comment:
`skipAccessCheckIfInterface` is now unused right? Can it be deleted?

Reply:
If we feel strongly that this PR should be a permanent change and not just a workaround, then we can delete it.

Reply:
I would delete it either way; it's easy to bring back if it were needed again, but until it's used again it's decoy code.

Reply:
deleted